### ISSUE TITLE
[: -eq: unary operator expected error encountered

### ISSUE THREAD

**Issue by knightsg** [kirkpatrickprice/linux-audit-scripts#22]
I get this running the script:

`./kpnixaudit.sh: line 2012: [: -eq: unary operator expected`

Appears to be because NOKUBES is undefined if the -k flag isn't passed in. Using double square brackets fixes this.

System Info:
```
OS: Ubuntu 20.04.6 LTS
Bash: GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
```

**Comment by flyguy62n** [comment]
This type of error is almost always due to running it though something over than the Bash shell.  Try opening a new Bash session and running it again.

**Comment by knightsg** [comment]
I'm running the script using sudo (eg. `sudo ./kpnixaudit.sh`) as from the documentation, so I changed to root using `sudo su -` and ran `/bin/bash --version` as that's what the script has in the shebang:

```
# /bin/bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

It appears that it's running under bash. As far as I can tell, the error is legit and is caused by the NOKUBES variable being unset earlier because the -k flag wasn't passed.

**Comment by flyguy62n** [comment]
I was able to get it to throw that message on my end and the fix was super-simple -- just declare the NOKUBES=0 variable ahead of time.  I actually don't know how it wasn't throwing that error all along.

Anyway, fixed now in v0.6.22 that just committed to GH.
